### PR TITLE
Detective gear.

### PR DESCRIPTION
**Pull request by DrSnips**
[content][powercreep]
People wanted detective gear changes to be it's own seperate PR so here it is.
- Added a belt to the detective's locker (in their gear box) that holds almost all of their usual gear like the scanner, binoculars, hand TV, evidence bags, fingerprint cards, etc. ~~This is a child of the sec belt so it does inherit all of the items from it. I'm open to changing this since it's literal power creep but since they are technically a sec role I felt like they should be allowed to have the gear on their belt if needed. **This does not mean they start with those items.**~~ Also holds glasses, ammo storage (speedloaders), donuts, cigarette packs, lighters, flashlights, pdas, headsets, handcuffs, flashes(not bangers), sec tape, hailer, sps, and holobadge.
- They get a spray bottle of luminol now which also fits on the belt.
- Other items added to the box: ~~tactical flashlight, boot knife~~, roll of sec tabe, and a switchblade/comb.

The switchblade has a knife with the same stats as a normal bootknife and comb which is a reskinned pocket mirror. Im aware the blade part doesn't look two sided but a knife sprite already exists for switchtools so it has to be.
switchblade

**OTHER STUFF**
- Made some items the parent rather than each individual child like .../donut/jelly and .../donut/normal just being .../donut now along with a few redundant item paths. This doesn't change how they function.
- Fixed a typo for telebatons in the allowed items list for sec belts.
:cl:
 * rscadd: Detectives now have a belt that can carry most of the gear from their locker and other job related items such as fingerprint cards. Also added a spray bottle of luminol, switchblade, and sec tape to their gear box.
* tweak: Detective scanners are now small. The mirror in the NT disguise kit are now combs and function the same way.

### CODE REVIEW

**Comment by AI-Is-Rogue** [comment]
The only thing I don't like is the boot knife, doesn't look good on shoes and is a crime against fashion.

File: code/game/objects/items/weapons/storage/belt.dm [hunk 1]
```diff
@@ -223,12 +221,35 @@
...
 		"/obj/item/weapon/depocket_wand",
 		)
+
+/obj/item/weapon/storage/belt/security/detective
+	name = "hard-worn belt"
+	desc = "There's a lot you can tell about a man from his clothes - sometimes it's all he can afford, or maybe he chooses to wear something as a message... this belt, then, is a statement. Classy, but not too drab. Fashionable, but still useful. People look at this belt and think, 'My god. That belt is frighteningly well placed. The shades of black that seem to flood over themselves, splayed across every notch... I fear many things, but I fear most the man who possesses such an incredible belt.'"
+	//fits_max_w_class = 3
+	max_combined_w_class = 200
+
+/obj/item/weapon/storage/belt/security/detective/New()
```
Comment (DamianX) [inline code/game/objects/items/weapons/storage/belt.dm]: Needs to call `..()`

**Comment by DrSnips** [comment]
@AI-Is-Rogue Its for holding your backup pen just in case you loose your pda's.

**Comment by SonixApache** [comment]
give belt a custom sprite and make it not a sec child
delet boot knife and tac-light

**Comment by DrSnips** [comment]
A bit of reason for those requests would be nice.

**Comment by soyconsumer** [comment]
Can you also add the NT disguise kit to the detective's locker on every map? It's currently hardmapped on meta

**Comment by DrSnips** [comment]
The kit was something to make up for the removal of the thermal eye patch iirc. Also it kind of sucks so its more of an exclusive map flavor thing that I'd rather not take away from meta.

**Comment by SonixApache** [comment]
1: Detectives don't get access to half the shit that fits on the belt
2: The belt holds things that security officers need. Detectives aren't security officers.

Boot knife and tac-light don't fit in the detective stereotype if you ask me.
If you want to give them something nice and fitting, make a cigarette case accessry or something. Binoculars holster. Whatever.

The NT diskit would be nice to have in the box, it's cute and it fits and it's on roid anyway because @GeneralVeers25 is a powercreeping dummy

**Comment by DrSnips** [comment]
How about I give them a zippo themed flashlight? Something a bit more fitting. The knife holster while not pretty is still a useful utility item and you aren't forced to wear them if you don't like how they look.
By all means detectives are a part of security and you cannot deny it. They have access to most of sec, they are mapped as sec, they are in the security section of manifest, they have a red ID, they have a red hud icon, and they report to the HoS. Stop trying to deny they aren't a part of security. The belt has the items if and when they are told to be more like an officer than an alcoholic. You can also open officer lockers Im pretty sure.

**Comment by Blithering** [comment]
Detectives start with pda flashlight enhancer, the taclight is not needed. While detectives are a part of sec, they do a different job in an ideal situation, and shouldn't start with officer gear lest they start thinking they're officers

**Comment by bathosbathtime** [comment]
Detectives are not officers and we should not encourage blurring the line between the two.

**Comment by DrSnips** [comment]
Alright I can remove it being a child but they do need some gear in order to detain people for questioning or whatever and yes this is part of thier design since their silly bullets revolver is clearly for stopping rather than killing.
Give me a list of what you do or don't want them to have.

**Comment by Blithering** [comment]
I'd say keep the belt holding forensics related stuff, so if there's really a need for the detective to play officer or nuke ops bust down the door he can ditch the whole thing and go grab a secbelt. Currently I do the same thing with the briefcase.

**Comment by SonixApache** [comment]
Can hold: Everything in the detective's locker / box
Cannot hold: anything else

Also, you're missing the point; detectives are not security *officers*; they are not intended to be the ones running around chasing bad guys, making arrests and fighting on the front line. They're expected to be the low-down evidence gatherer, the snooper, the maint-hole-investigating researcher.
The revolver is more for self defense than detaining people, I'd say.

**Comment by DrSnips** [comment]
So what you're saying is add all sec gear to the detective's locker :^)
Jokes aside if you don't give me an actual list I'm going to present you with my limited list. Which is the current additions for the det belt, most flavor items(lighters, donuts, glasses, etc), flashes, handcuffs, and some other things. Some notable exclusions are stun batons and tasers. This way they still have the ability to restrain not so compliant suspects for questioning. I feel like they should have the ability to detain but not do complete arrests. I know you would like them to be completely unable to do anything but gather scans but the fact is that you're dealing with antags and other potentially dangerous individuals which need more than the detective's silly bullets.

**Comment by SonixApache** [comment]
detectives with handcuffs are a horrible type of people

**Comment by Blithering** [comment]
I would restrict any combat related items like cuffs and flashes, not because detectives should never have them, but so they won't have to fish those out of the belt when swapping over in emergencies. Lighters, cigarette packets and other flavour items sound fine, hip flasks and evidence bags too.

**Comment by Blithering** [comment]
Also if you're feeling up for making new things, a switchblade or folding blade that's pocket sized when collapsed would be much more thematic for detectives than boot knives

**Comment by FudgePucker** [comment]
What about a Maglight? It could have boosted brute damage (have you ever held a D battery sized maglight? Things a club), and maybe bigger light radius. Would fit better for a detective than the tac light and be a smashing tool to make up for the lack of stunbatton, to get into baricated areas.

**Comment by DrSnips** [comment]
Alright I can do a switchblade. As for the flashlight I can make one with decent force if people would rather have that. Just some numbers for lights and their ranges lighters/pda:2, pda upgraded:3, flashlights:4.

**Comment by DrSnips** [comment]
Added the switchblade and comb. A themed knife with a comb option as well for fun. The knife inside is the same as sec knifes. Combs are reskinned pocket mirrors and replace the mirror inside the disguise kit.
I have removed them being a child of sec belts but I have carried over a few items I felt fit their job **IMO** or are flavor items. The belt is now brown rather than black. My current list of gear is now:
- all of the actual detective gear from before like cameras, folders, scanner, etc.
- from the sec belt: glasses, ammo storage (speedloaders), donuts, cigarette packs, lighters, flashlights, pdas, headsets, handcuffs, flashes(not bangers), sec tabe..., hailer, sps, holobadge.
- new item: the vamp head item from vox trader's sec selection and switchblade.

The detective scanner is now a small item due to size issues with the belt. 100% tested.
Removed the flashlight and bootknife from the detective's box, added the switchblade.

**Comment by DrSnips** [comment]
If you have further issues with the gear do what you have been by yelling at me and giving me downboats.

**Comment by SonixApache** [comment]
all good minus the cuffs in the belt

**Comment by DrSnips** [comment]
Curse you sonix and your byond memes.

**Comment by Blithering** [comment]
Looking good, the switchblade/comb combo sounds neat, while I'd still advise against cuff and flash storage its not a hill I'm gonna die on.

**Pull request merged**